src/pages/workspaces/workspace/analysis/modals/HailBatchModal.js

```diff
@@ -0,0 +1,151 @@
+import _ from 'lodash/fp';
```

Comment:
would be nice to have some tests around this added functionality.

The following seems warranted: 
1) base modal render works
2) create API is called when button it clicked
3) deleteWarn message is shown after initial delete click 
4) delete api is called when button is clicked

I also highly encourage you to make this new file with a `.ts` extension! You might have to ignore some errors stemming from common components/lodash, but it will provide a large amount of protection when using our already-typed utility functions

Reply:
added test and converted to ts!